frontend/package.json

```diff
@@ -82,6 +91,7 @@
     "downshift": "^1.22.5",
     "find-parent": "^2.0.2",
     "highlight-words-core": "^1.1.2",
+    "jest": "^22.1.4",
```

Comment:
do we really need this as a prod dependency?

Reply:
👍